randovania/games/prime1/json_data/Phendrana Drifts.txt

```diff
@@ -2397,6 +2426,19 @@ Extra - aabb: [329.07877, -1248.0734, -17.693329, 398.72052, -1143.9733, 42.7440
                       Slope Jump (Advanced)
                   Scan Visor and Combat/Scan Dash (Advanced)
           Shoot Plasma Beam and Use Grapple Beam
+          All of the following:
+              After Gravity Chamber Item (Lower)
+              Any of the following:
+                  All of the following:
+                      Charge Beam
+                      Any of the following:
+                          Space Jump Boots and Jump Off Enemies (Advanced)
+                          All of the following:
+                              # https://youtu.be/V_CBSW0l8Hs
```

Comment:
While this is better than nothing rule 7 of the video submission guidelines states

> The game must be running at full speed (e.g. Dolphin must maintain ~60fps)

Reply:
This is currently the only footage of this trick being performed, and while I would recreate it in higher quality, I'm very inefficient at replicating the trick, and would rather update with a new video in a different PR.